frontend-react/src/styles/_reportstream.scss

```diff
@@ -771,4 +771,21 @@ pre {
 
 .rs-margin-top-auto-important {
     margin-top: auto !important;
-}
\ No newline at end of file
+}
```

Comment:
Oh my....this might be my first time seeing this file, but the indentation and styling is ALL over the place. Might be a good candidate to make a separate ticket to clean this up? Way too many specific styles going on here (not your fault, was going to comment on that on this code, but saw it's throughout this file).